src/stylesheets/components/_table.scss

```diff
@@ -139,6 +139,15 @@ $sds-table-border: "1px";
 
 .usa-table {
 
+  &.full-width {
+    width: 100vw;
```

Comment:
It appears that width: 100% would be sufficient to cause the table to take up all width available to it. Is there a particular reason that approach wouldn't work?